src/java.base/share/classes/java/util/DoubleSummaryStatistics.java

```diff
@@ -156,7 +156,9 @@ public void combine(DoubleSummaryStatistics other) {
         count += other.count;
         simpleSum += other.simpleSum;
         sumWithCompensation(other.sum);
-        sumWithCompensation(other.sumCompensation);
+
+        //Negating this value because low-order bits are in negated form
+        sumWithCompensation(-other.sumCompensation);
```

Comment:
Wouldn't that be `double tmp =  sum + sumCompensation;` in getSum() in line 246 too?

Reply:
Good catch will review and make the change.